packages/appstore/src/components/routes/routes.tsx

```diff
@@ -10,10 +11,13 @@ import RouteWithSubroutes from './route-with-sub-routes.jsx';
 
 const Routes: React.FC = observer(() => {
     const { is_wallet_enabled } = useFeatureFlags();
-    const { has_wallet } = useWalletsList();
+    const { has_wallet, isLoading } = useWalletsList();
```

Comment:
@heorhi-deriv if you are using useWalletMigration() hook, in this case I suppose we don't need to use useWalletsList(), because we can check state === migrated (or just get is_migrated from useWalletMigration). Just a suggestion, you can ignore it if you want.